ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-stable-4.18-upgrade-from-stable-4.17.yaml

```diff
@@ -42,18 +42,17 @@ build_root:
     tag: golang-1.18
 releases:
   latest:
-    release:
-      architecture: multi
-      channel: candidate
```

Comment:
Why we need to change the release image setting ?

Reply:
To be consistent with [4.16-4.17 stable upgrade](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-stable-4.17-upgrade-from-stable-4.16.yaml#L44-L50) and [4.15-4.16 stable upgrade job](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-stable-4.16-upgrade-from-stable-4.15.yaml#L44-L50).